resources/bigtop-1.1.0/BIGTOP-2490.patch

```diff
@@ -24,97 +22,36 @@ index f5a2d64..0fd0603 100644
        $master_port = 7077,
        $worker_port = 7078,
        $master_ui_port = 8080,
-@@ -158,13 +159,24 @@ class spark {
+@@ -158,6 +159,13 @@ class spark {
        ensure => latest,
      }
  
 +    if $zookeeper_connection_string {
 +      $spark_daemon_java_opts = "\"-Dspark.deploy.recoveryMode=ZOOKEEPER -Dspark.deploy.zookeeper.url=${zookeeper_connection_string}\""
-+      $spark_env = "spark/spark-env.ha.sh"
-+      $spark_defaults = "spark/spark-defaults.ha.conf"
 +    }
 +    else {
 +      $spark_daemon_java_opts = "\"-Dspark.deploy.recoveryMode=NONE\""
-+      $spark_env = "spark/spark-env.sh"
-+      $spark_defaults = "spark/spark-defaults.conf"
 +    }
 +
```

Comment:
It the previous (and unsuccessful) attempt to introduce HA I thought I would have two sets of spark-*.conf scripts (one for HA and one for non-HA). I realised this is not really needed since the one option that is non-HA specific can be feature-flagged. So, the above code used to select if it was going to use the spark-*.ha.confs , now we always use the same template files.